CONTRIBUTING.md

```diff
@@ -55,13 +55,19 @@ The `changelog.py` script is used to generate the final changelog (it's not perf
 - Use _rebase workflow_ for all PRs.
 - When addressing review comments, if possible, update original commits with fixes belonging to them. Then force-push.
 
+## Naming Conventions
+- Type names are in `PascalCase`.
+- Global constants and enum items are in `SCREAMING_CASE`.
+- Variables (local, global, struct members) and functions are in `snake_case`.
+- File names are in `snake_case.c`.
+
```

Comment:
I liked the place naming conventions were before. Here you've put them inbetween Pull Requests and CI, I'm not sure how that fits logically...

Reply:
Moved it one "spot" lower, before code formatting. I think it logically fits there, but I suppose it could stay down at the bottom as well. Either way is fine with me, would you prefer it at the bottom?

Reply:
Honestly I prefer it where it was, because the Code Formatting section below is about the `clang-format` setup, which is part of setting the environment. The naming conventions is when you go edit the code by hand, and that to me fits better after the whole environment setup part. So I'd keep it at the end.